src/rshim.c

```diff
@@ -2587,8 +2588,17 @@ int rshim_access_check(rshim_backend_t *bd)
       bd->in_access_check = 0;
       return -EEXIST;
     }
+  }
 
-    usleep(100000);
+  // Write RSHIM_KEEPALIVE_MAGIC_NUM to SP1 in case the other backend is also
+  // running the previous checking loop. This magic number isn't otherwise
+  // written during this function by the timer handler due to in_access_check.
+  rc = bd->write_rshim(bd, RSHIM_CHANNEL, bd->regs->scratchpad1,
+      RSHIM_KEEPALIVE_MAGIC_NUM, RSHIM_REG_SIZE_8B);
```

Comment:
Can we add space to align the 'RSHIM_KEEPALIVE_MAGIC_NUM' with the 'bd'? Most of other places tries to use this style I think.

Reply:
Changed.